From akpm@linux-foundation.org Mon Jul  9 12:04:10 2007
From: Jan Engelhardt <jengelh@computergmbh.de>
Date: Mon, 09 Jul 2007 12:03:06 -0700
Subject: USB: fix gregkh-usb-usb-use-menuconfig-objects
To: greg@kroah.com
Cc: linux-usb-devel@lists.sourceforge.net, akpm@linux-foundation.org, jengelh@computergmbh.de, jens.axboe@oracle.com
Message-ID: <200707091903.l69J36Cx002778@imap1.linux-foundation.org>


From: Jan Engelhardt <jengelh@computergmbh.de>

Generally, Jens Axboe was against 'default y', so I'll have some patches to
remove it.

Cc: Jens Axboe <jens.axboe@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/usb/Kconfig
+++ b/drivers/usb/Kconfig
@@ -5,7 +5,6 @@
 menuconfig USB_SUPPORT
 	bool "USB support"
 	depends on HAS_IOMEM
-	default y
 
 if USB_SUPPORT
 
